search/src/main/scala/weco/api/search/services/SearchService.scala

```diff
@@ -61,11 +61,15 @@ trait SearchService[T, VisibleT, Aggs, S <: SearchOptions[_, _, _]] {
     searchOptions: S,
     index: Index
   ): Future[Either[ElasticsearchError, SearchResponse]] = {
-    val searchRequest = requestBuilder
+    val request = requestBuilder
       .request(searchOptions, index)
-      .trackTotalHits(true)
     Tracing.currentTransaction.addQueryOptionLabels(searchOptions)
-    elasticsearchService.executeSearchRequest(searchRequest)
+    request match {
```

Comment:
Again, this Either/match business is here to allow me to leave old-style functionality in place and finish this work in stages.